ddouglasz/pokemon
# Feat/get all pokemons

**What does this PR do?**

- Fetch Pokemon Data

 **Description of Task to be completed?**

 - PR adds feature for user to see a pokemon summary and at least a picture of a pokemon.

 **How should this be manually tested?**
- get the app up and running: `yarn dev`
 - navigate to localhost:300 and ensure it display's at up to 16 pokemon characters

 **Any background context you want to provide?**

 xxx

 **What are the relevant stories?**

 xxx

 **Screenshots (if appropriate)**

 xxx

 **Questions:**

xxx
